sopel/coretasks.py

```diff
@@ -825,6 +919,17 @@ def send_authenticate(bot, token):
 @module.event('AUTHENTICATE')
 @module.unblockable
 def auth_proceed(bot, trigger):
+    """Handle client initiated authenticate.
```

Comment:
```suggestion
    """Handle client-initiated SASL auth.
```

AFAIK, SASL is the only use for `AUTHENTICATE` in IRC specs. Did I miss something?